ios/browser/api/ledger/brave_ledger.mm

```diff
@@ -543,7 +543,7 @@ - (void)fetchBalance:(void (^)(LedgerBalance* _Nullable))completion {
 - (void)recoverWalletUsingPassphrase:(NSString*)passphrase
```

Comment:
`recoverRewardsWalletUsingPassphrase`?

Reply:
I'm shy about making any changes to the iOS APIs...

Reply:
Usually I just kick off a build [here](https://ci.brave.com/view/ios/job/test-brave-browser-build-ios) with my branch to make sure I don't break iOS folks, but in this case, there are no references to `recoverWalletUsingPassphrase` in `brave-ios`. Actually, there are no references to it in `brave-core` either, so instead of renaming it, we should just remove it. 🙂 cc: @Brandon-T

Reply:
I think I would prefer not to make changes to iOS APIs in this PR - overall it looks like it might need some cleanup.

Reply:
Usually we remove the wrapper definitions of methods when the underlying C++ method is also removed (`RecoverWallet` in this case)

Despite it no longer being used in iOS—we don't really have a lot of rewards functionality lol—it's easier to wrap things and have them in the framework in case we need them for some reason rather than hit a point where we need it and have to go write the wrapper and uplift it across versions to get it in iOS.